### DISCUSSION TITLE
KDP Estimation method

### DISCUSSION THREAD

**Discussion by saranya00-s** [uobwatergroup/towerpy#69] [Q&A] [answered]
Hi. Which method is used to estimate KDP. Please clarify

**Comment by enchilaDaSzen** [comment, marked as answer]
Hi @saranya00-s , $$K_{DP}$$ is an output of the zh_correction function (AttenuationCorrection module). It is computed using the equation $$K_{DP}=A_H/\alpha$$; we consider you can rely on this  $$K_{DP}$$ estimation method given that  $$\alpha$$ is optimised ray-wise, e.g., by using the adaptive method (ABRI) implemented in this function. Hope this helps!
